src/web/topic/components/Indicator/DeleteIndicator.tsx

```diff
@@ -0,0 +1,26 @@
+import { Delete } from "@mui/icons-material";
+
+import { deleteEdge, deleteNode } from "../../store/createDeleteActions";
+import { Indicator } from "./Indicator";
+
+interface Props {
+  graphPartId: string;
+  graphPartType: string;
+}
+
+export const DeleteIndicator = ({ graphPartId, graphPartType }: Props) => {
```

Comment:
if this is a user's topic, we'll want to restrict this functionality from those that aren't the topic's creator, like this https://github.com/amelioro/ameliorate/blob/9f2f1d9f571b51ecb14183c418e2a33eb3d6921d/src/web/topic/components/Surface/TopicToolbar.tsx#L87-L97